app/shell-window/ui/navbar.js

```diff
@@ -679,6 +691,19 @@ function onClickGotoDatVersion (e) {
   }
 }
 
+function onClickGotoHttpVersion (e) {
+  const page = getEventPage(e)
+  if (!page || !page.protocolInfo) return
+
+  const url = `https://${page.protocolInfo.hostname}${page.protocolInfo.pathname}`
+  pages.noRedirectOrigins.add(url)
```

Comment:
We should do these by origin or hostname (hostname might be easier). If we're disabling the dat redirect, we really want to disable for all pages on that location.

Reply:
Agreed.